src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for Roboto in case it fails to load. For example, 'Roboto, Arial, sans-serif'.